README.md

```diff
@@ -1,12 +1,53 @@
+* Javascript
+* Documentation
+* Test driven development (TDD)
+* Jest
+* Supertest
+* Express
+* Model view controller (MVC)
+* PostgreSQL
+* Fs/promises
+* Hosting using supabase for the database and render for the website
 
-In order to run this repo locally, once forked and cloned you will have to add 2 .env files to the root of the cloned repo, these files should be named .env.development and .env.test
+### Access online
+To access this API online click on the following URL: [News API](https://news-api-ovyc.onrender.com/)
 
-In the .env.development file you will need the following line PGDATABASE=nc_news and in .env.test you will need PGDATABASE=nc_news_test. 
+For a list of endpoints, simply type /api in the url after .com
 
-After adding these 2 files the repo should run locally without issue.
+To access each endpoint, you also type it in the URL after .com
 
-hosted api url = https://news-api-ovyc.onrender.com/
---- 
+### Set up
+To interact with this api locally you will need to carry out the following instructions:
+1) in your terminal run the following command in your terminal type git ```clone https://github.com/TheStringKing/News-API.git``` . This will clone the news-API repository onto your local system.
+ 
+2) Next open the repository in a code editing software, for example visual studio code. In the __tests__ directory you will be able to see the dest driven development used to develop the API endpoints.
 
-This portfolio project was created as part of a Digital Skills Bootcamp in Software Engineering provided by [Northcoders](https://northcoders.com/)
+3) Next in the root of the repository create two files, one called .env.test and one called .env.development. In the .env.test file on line one type ```PGDATABASE=nc_news_test``` and in the .env.development file on line one type ```DATABASE_URL=postgresql://postgres.byfadzxrcidgzoyhgceo:KdjWqk2kr3E259f8LL7F8G3f7@aws-0-eu-west-2.pooler.supabase.com:6543/postgres```
+
+4) check that you have node and node project manager (npm) installed by run the commands ```node -v``` and ```npm -v``` in your terminal, if it does not respond with a version number for either you can install node [here](https://nodejs.org/en/download/prebuilt-installer/current) and install npm by running ```npm install -g npm```
+
+5)check that you have psql installed by ```running psql --version``` if you do not you can find instructions to do so [here](https://www.postgresql.org/download/)
+
+6) next you will need to install the following dependencies:
+| dependency  | code to run in your terminal |
+| ------------- |:-------------:|
+| express  	| ```npm install express``` |
+| dotenv  	| ```npm install dotenv``` 	|
+| pg  	| ```npm install pg``` 	|
+| pg-format  	|   ```npm install pg-format```   |
+| jest  	|   ```npm install jest```   |
+| jest sorted 	|  ```npm install jest-sorted```	|
+| supertest 	|   ```npm install supertest```   |
```

Comment:
these should be listed separately as dependencies. Also these are listed in the package.json, meaning that just running `npm i` will automatically install all the listed dependencies